src/docfx/config/ConfigLoader.cs

```diff
@@ -67,7 +69,7 @@ where glob(configPath)
 
             // Preload
             var preloadConfigObject = new JObject();
-            JsonUtility.Merge(preloadConfigObject, envConfig, globalConfig, opsConfig, docfxConfig, cliConfig);
+            JsonUtility.Merge(unionProperties, preloadConfigObject, envConfig, globalConfig, opsConfig, docfxConfig, cliConfig);
```

Comment:
```suggestion
            JsonUtility.Merge(unionProperties, preloadConfigObject, envConfig, globalConfig, opsConfig, docfxConfig, options?.StdinConfig, cliConfig);
```

Reply:
would prefer to put CLI config separately